runtime/dom-particle.js

```diff
@@ -99,9 +95,14 @@ class DomParticle extends XenStateMixin(Particle) {
     this.handles = views;
     this._views = views;
     let config = this.config;
-    this.when([new ViewChanges(views, config.views, 'change')], async () => {
-      await this._handlesToProps(views, config);
-    });
+
```

Comment:
Unrolling the changes code into this module seems like a step backward, what's the value add?

Reply:
If ViewChanges are going to be of use elsewhere/in the future, then I agree that we shouldn't do this. If it ends up being an API that only DomParticle needs, then the logic should live in DomParticle.

Ultimately I was trying to simplify the Particle code - pruning stuff. Apologies if it's too early to judge that.